USER-TRAINING.md

```diff
@@ -0,0 +1,33 @@
+# User training
+
+This document is for Ops team members setting up new developers to work in the GPII cloud.
+
+## The Speech
+
+* You are about to be granted administrative (aka "root") access to a number of resources on the public internet, including our staging and production environments. This is a serious responsiblity! You will be able to do harm to the system!
+* Measure twice, cut once.
+* If you're ever in doubt about what a command will do, STOP! Then ask for help.
+* Be ethical. Don't steal data. Don't run your startup on RtF's billing account.
+* RtF pays for these resources so please destroy your cluster when you're done using it, or won't use it for a while (e.g. before the weekend).
+
+## Overview of the system
+* Go over the [Build and Release Overview slide deck](https://docs.google.com/presentation/d/1l8qQEvFaml_qgc0fynHScVhWseu0loytcYaFP_m0tBs/edit#slide=id.g3150fb0231_0_0) with the user.
+   * Remember that many product developers don't have a lot of experience with infrastructure tools and concepts. Introduction of high-level concepts like containers, container orchestration, and automated deployment as well as a general view of how the pieces fit together is the goal; the fine details are less important.
+
+## AWS: Creating the user's IAM
+* From the AWS Dashboard, navigate to IAM controls (under Services or use the search box)
+* Add User
+   * Naming convention is "FirstLast", e.g. "TylerRoscoe"
+   * No to "Programmatic access" (User will create an Access Key later)
+   * Yes to "AWS Management Console access"
+   * Choose "Autogenerated password"
+   * Yes to "User must create a new password at next sign-in"
+* Add User to group
+   * FullAdmin
```

Comment:
Looks like there used to be a different set of permissions before.
Here is what I see for my user:
https://www.dropbox.com/s/sy7yvm7kulcsa1f/Screenshot%202018-07-16%2018.38.48.png?dl=0
Why we decided to use `FullAdmin` perms?

Reply:
Perhaps the "Access advisor" can help:

https://console.aws.amazon.com/iam/home?region=us-east-2#/users/SteveGithens?section=access_advisor

I choosed Steve's user because is a brand new developer user, who was able to spin up a complete cluster.